src/ui/Base/BaseComponent.ts

```diff
@@ -22,8 +22,8 @@ export class BaseComponent {
    */
   public disabled = false;
   /**
-   * The static ID that each component need to be identified.<br/>
-   * For example, SearchButton -> static ID : SearchButton -> className : CoveoSearchButton
+   * The static ID that each component needs to be identified.<br/>
```

Comment:
`needs to be identified with` ?

Reply:
I guess it depends: is it the static ID that each component needs in order to be identified? Or is it the static ID that identifies each component?

Reply:
Hmm the first, could we add `in order` in the string, would sound more clear a bit

Reply:
Another option could be to reorder the words: `The static ID that identifies each component`.

@jgeddes-coveo, both those phrasings seem similar, I'm not catching the nuance? This property has value `Tab` for the tab component, `Facet` for the facet component, and so on.